user_sync/connector/directory_ldap.py

```diff
@@ -197,6 +232,32 @@ def find_ldap_group_dn(self, group):
                 group_dn = current_tuple[0]
         return group_dn
 
+    def find_group_member_dn(self, group, member_attribute):
+        """
+        :type group: str
+        :type member_attribute: str
+        :rtype list
+        """
+        connection = self.connection
+        options = self.options
+        base_dn = six.text_type(options['base_dn'])
+        group_filter_format = six.text_type(options['group_filter_format'])
+        try:
```

Comment:
I believe this whole function is wrapped in a try: except: further up.  I don't think you want one here as well.  Was this cribbed from find_group_dn?